<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Azael Avalos &lt;coproscefalo@gmail.com&gt;</author>
 <class>Keyboard</class>
 <widget class="QWidget" name="Keyboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Keyboard Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="function_keys_label">
        <property name="text">
         <string>Function Keys mode:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="function_keys_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="kbd_functions_combobox">
        <property name="toolTip">
         <string>Select how the keyboard Function keys operate.  Normal: The F{1-12} keys are as usual and the hotkeys are accessed via FN-F{1-12}.  Function: The F{1-12} keys trigger the hotkey and the F{1-12} keys are accessed via FN-F{1-12}.</string>
        </property>
        <property name="whatsThis">
         <string>Select how the keyboard Function keys operate.  Normal: The F{1-12} keys are as usual and the hotkeys are accessed via FN-F{1-12}.  Function: The F{1-12} keys trigger the hotkey and the F{1-12} keys are accessed via FN-F{1-12}.</string>
        </property>
        <item>
         <property name="text">
          <string>Normal</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Function</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="kbd_backlight_label">
        <property name="text">
         <string>Keyboard Backlight mode:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <spacer name="kbd_backlight_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item row="1" column="2">
       <widget class="QComboBox" name="kbd_backlight_combobox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="kbd_timeout_label">
        <property name="text">
         <string>Keyboard Backlight timeout:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <spacer name="kbd_timeout_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" alignment="Qt::AlignRight">
       <widget class="QLabel" name="kbd_timeout">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>The current timeout at which the keyboard backlight will turn off after the last keypress.</string>
        </property>
        <property name="whatsThis">
         <string>The current timeout at which the keyboard backlight will turn off after the last keypress.</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QSlider" name="kbd_timeout_slider">
        <property name="toolTip">
         <string>Sets the time the keyboard backlight will turn off after the last keypress.</string>
        </property>
        <property name="whatsThis">
         <string>Sets the time the keyboard backlight will turn off after the last keypress.</string>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>60</number>
        </property>
        <property name="pageStep">
         <number>1</number>
        </property>
        <property name="sliderPosition">
         <number>15</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksAbove</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
